packages/bodiless-documentation/doc/Development/Guides/DesignElementConcepts.md

```diff
@@ -1,329 +1,327 @@
+        Link: as(mysiteElement.Link, asBodilessLink()),
+      },
+    });
+
+    export default {
+      Simple,
+    };
+    ```
+
+01. Export the tokens from the package as you did before for `mysiteElement`:
+
+    01. `/packages/mysite/src/components/RichText/tokens/index.ts`:
+
+        ```ts
+        import mysiteRichText from './mysiteRichText';
 
-First, let's look at the `mysiteRichText` token collection. It contains a single
-token, `Simple`, expressed as a *token specification*. A token described in this
-notation is a 2-level nested object. The inner keys of this object are simply
-the design keys of the component to which the token applies, and their values
-are higher-order components which should be applied to each element. The top
-level keys represent "domains" -- pieces of the token which might be selectively
-reused, extended or overridden by a downstream consumer. Here we are using just
-a single domain, `Core`. We'll show later on how to define and use multiple
-domains to facilitate extension and customization.
-
-In this case, the design keys represent the components which are available in
-the editor to render different text formats, as described in [the first tutorial](../IntroToBodilessConcepts#6-configuring-the-rich-text-editor). We apply our
-typography tokens to each, thus ensuring that any changes will propagate to all
-simple rich text editors on the site.
-
-Note the special design key `_`. Tokens supplied in this key apply to the editor
-as a whole. Here we make the editor `stylable` -- that is capable of receiving
-classes via the FClasses API. We alo supply a placeholder to be used as a
-default, though this will usually be overridden where the editor is used. This is
-because the placeholder--like the node key--is usually defined by the context
-in which the token is placed, and therefore not reusable.
+        export default mysiteRichText;
+        ```
+
+    01. `/packages/mysite/src/components/RichText/index.ts`:
+
+        ```ts
+        import mysiteRichText from './tokens';
+
+        export { mysiteRichText };
+        ```
+
+    01. `/packages/mysite/src/index.ts`:
+
+        ```ts
+        export * from './components/RichText';
+        ```
+
+01. Rebuild your package (execute `npm run build` from the package directory or `npm run
+    build:packages` from outside it), and use these tokens in the Gallery.
+
+    01. In `/sites/mysite/src/data/pages/gallery/index.tsx`, add these imports:
+
+        ```ts
+        import { withNodeKey } from '@bodiless/core';
+        import { RichText } from '@bodiless/richtext-ui';
+        import { withPlaceholder } from '@bodiless/components';
+        import { mysiteElement, mysiteRichText } from 'mysite';
+        ```
+
+        Then change:
+
+        ```ts
+        const Body = withSimpleEditor('body', 'Body')(Div);
+        ```
+
+        to:
+
+        ```ts
+        const Body = as(
+          mysiteRichText.Simple,
```

Comment:
Just to provide another example, this token — like `mysiteElement.H1` previously mentioned — also generates the TypeScript error:

`Argument of type 'TokenSpec<DesignableComponents, { Core: {}; Schema: {}; Layout: {}; Theme: {}; Meta: {}; Editors: {}; }>' is not assignable to parameter of type 'Token'....`